src/app/manager-dashboard/reports/reports-detail-activities.component.ts

```diff
@@ -23,17 +24,27 @@ export class ReportsDetailActivitiesComponent implements OnInit, OnChanges, Afte
   constructor() {}
 
   ngOnInit() {
+    this.displayedColumns = this.displayedColumns.concat(this.activityType === 'courses' ? [ 'enrolled', 'passed' ] : []);
     this.activities.sortingDataAccessor = (item: any, property: string) =>
       sortNumberOrString(this.sortingObject(item, property), property);
   }
 
   ngOnChanges() {
-    this.activities.data = this.activitiesByDoc.map(
-      activity => ({
+    this.activities.data = this.activitiesByDoc.map(activity => {
+      const progressCount = { enrolled: 0, passed: 0 };
+      /* if (this.activityType === 'courses') {
+        const progress = (this.progresses.find(p => p.doc._id === activity.courseId) || { progress: {} }).progress;
```

Comment:
1. Use `progresses.filter` to get only the progress documents that are for that course
2. `enrolled` should be the number of documents for step 1 (steps should be unique per user)
3. `completed` (use that instead of passed) should be for the number who passed all steps.  You can use `CertificationsService.isCourseCompleted` for that